### DISCUSSION TITLE
Tabs using Collapse

### DISCUSSION THREAD

**Discussion by LuziferSenpai** [saadeghi/daisyui#2173] [Ideas / feature requests]
Hey,

isnt it possible to make tabs now using collapse?
That would make using JS useless and makes using DaisyUI even better.

Greetz,
Luzifer

**Comment by saadeghi** [comment]
It is possible to make that.  
I will look into that when I get free time but currently I need to focus on fixing issues.

**Comment by LuziferSenpai** [comment]
Hey, just wanted to know if there is any update on this or will not come soon?

**Reply by saadeghi** [reply]
It will be in daisyUI 4. Coming in 2~3 weeks
